ephios/core/templates/core/event_list.html

```diff
@@ -128,21 +112,19 @@ <h3 class="mx-3">
                     <i class="fas fa-arrow-right"></i></a>
             </div>
         {% endif %}
-
-
     </div>
 
-    {% if mode == "calendar" %}
-        {{ calendar }}
+    {% if mode == "list" %}
+        {% include "core/fragments/event_list_list_mode.html" %}
     {% elif not event_list %}
```

Comment:
this would hide the calendar if there are no results, is that really what we want?

Reply:
nope